main.tf

```diff
@@ -3,29 +3,21 @@ This file is used to implement the ROOT module.
 *********************************************************************/
 
 resource "ibm_db2" "db2" {
-  name              = var.db2_instance_name
-  resource_group_id = var.resource_group_id
-  location          = var.region
-  plan              = var.subscription_id == null ? "performance" : "PerformanceSubscription"
-  service           = "dashdb-for-transactions"
-  high_availability = var.enable_high_availability ? "yes" : "no"
-  service_endpoints = var.service_endpoints
-
-  parameters = {
```

Comment:
need to know what impact that has on the module since you skipped the upgrade test. Is it an update in place, or destroy / re-create

Reply:
Updated the description with the results of a manual upgrade test, updates in place